java-client/build.gradle.kts

```diff
@@ -151,7 +151,9 @@ dependencies {
     compileOnly("org.opensearch.client", "opensearch-rest-client", opensearchVersion)
     testImplementation("org.opensearch.test", "framework", opensearchVersion)
 
-    api("org.apache.httpcomponents.client5:httpclient5:5.2.1")
+    api("org.apache.httpcomponents.client5:httpclient5:5.2.1") {
+      exclude(group = "org.apache.httpcomponents.core5")
```

Comment:
@reta if we are excluding the dependency on core here. Do we still need to exclude it on the SDK repo?

Reply:
@owaiskazi19  yes, because for SDK, there are indeed 2 clients (rest & this one), both importing Client5 / Core5, the 3.0.0 release line has no runtime dependencies on OpenSearch core. When SDK picks just one client - issue will be gone.